pkg/synchronizer/kubernetes/sync_server.go

```diff
@@ -191,35 +175,11 @@ func CreateSynchronizer(config, remoteConfig *rest.Config, scheme *runtime.Schem
 	return s, nil
 }
 
-// Start caches, this will be triggered by the manager.
+// this will be triggered by the manager.
 func (sync *KubeSynchronizer) Start(ctx context.Context) error {
```

Comment:
This `Start` function is required for the controller to start, but since we removed the cached client it doesn't do much, any suggestions for it or is it fine how it is?

Reply:
@xiangjingli  Oh I tried removing it but I can't remove the whole function as I get the error, `Synchronizer does not implement manager.Runnable (missing method Start)`. I can either make it return nil or keep it how it is, it shouldn't impact anything else, what do you think?

Reply:
OK, then just leave it there. It doesn't impact anything :-)